src/Cryptol/Eval/Concrete/Value.hs

```diff
@@ -313,9 +319,43 @@ instance Backend Concrete where
   znNegate _ 0 _ = evalPanic "znNegate" ["0 modulus not allowed"]
   znNegate _ m x = pure $! (negate x) `mod` m
 
+  ------------------------------------------------------------------------
+  -- Floating Point
+  fpLit                  = FP.fpLit
+  fpEq _sym x y          = pure (FP.bfValue x == FP.bfValue y)
+  fpLessThan _sym x y    = pure (FP.bfValue x <  FP.bfValue y)
+  fpGreaterThan _sym x y = pure (FP.bfValue x <  FP.bfValue y)
```

Comment:
Looks like `<` should be `>` here

Reply:
Thanks at @robdockins!.    Hey could you have a look at module `Cryptol.Eval.What4.SFloat`?  I tried to keep it independent of Cryptol, so that we can move it to `what4` eventually.  I tried to mirror the design of `SWord`, but let me know if you think it should be done in another way.